load_data.py

```diff
@@ -10,31 +10,37 @@
 from django.core.management.sql import sql_delete
 from django.db import connections, DEFAULT_DB_ALIAS, models
 from django.db.utils import IntegrityError
+from django.core.exceptions import ValidationError
 
 from data.parse_document import parse_document
 from data.parse_schema import parse_schema
 from parse_schema import TopicsSchemaParser
 from thresher.models import Article, AnalysisType, TUA, Topic
-
+import pprint;
 ANALYSIS_TYPES = {}
 HIGH_ID = 20000
 
 def load_schema(schema):
+    print(schema['title'])
+    print(schema['parent'])
     schema_name = schema['title']
-    schema_obj = AnalysisType(
+    schema_parent = schema['parent']
+    # if schema_parent:
+    #     p_id = Topic.objects.get(name=schema_parent).id #TODO: Query for parent.
+    # else:
+    p_id = None
+    schema_obj = Topic(
+        parent = p_id,
         name=schema_name,
-        requires_processing=schema_name not in ['Useless', 'Future'],
```

Comment:
@nickbadams @FadyShoukry 
What is the purpose of `requires_processing` in `analysis_type` (now topic)?